meross_iot/supported_devices/power_plugs.py

```diff
@@ -409,4 +409,20 @@ class Mss210(Device):
 
 
 class Mss110(Device):
```

Comment:
If this applies only for newer versions, we should add version checks. Please provide more info about your device version  first.

Reply:
The firmware I am using is 2.1.12. I'm glad to add firmware information. It seems there is no firmware parameter yet. How would you like to add it?

Reply:
At the end, I've decided to refactor the library from scratch. Now it should be firmware/hardware version agnostic!

Reply:
Good point! Let me know how you'd like to move forward. Cheers.